core/ustring.cpp

```diff
@@ -3049,6 +3049,23 @@ String String::replacen(const String &p_key, const String &p_with) const {
 	return new_string;
 }
 
+String String::repeat(int p_count) const {
+
+	ERR_FAIL_COND_V_MSG(p_count < 0, "", "Parameter count should be a positive number.");
```

Comment:
Let me know if this should be removed. There is error check in resize also.

Reply:
The check is good, as is the default value in case of failure.